about.html

```diff
@@ -0,0 +1,146 @@
+            alt=""
+          />
+          <div class="logo">
+            <img src="./images/main-logo.png" alt="logo" />
+          </div>
+          <ul class="nav_menu d-flex">
+            <li class="menu_link"><a href="./index.html">Home</a></li>
+            <li class="menu_link active">
+              <a href="#About">About us</a>
+            </li>
+            <li class="menu_link"><a href="#">Program</a></li>
+            <li class="menu_link"><a href="#">Team</a></li>
+            <li class="menu_link"><a href="#">Partners</a></li>
+            <li class="menu_link active">
+              <a href="./index.html">Our Gorillas</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <main>
+      <section id="About" class="home_section">
+        <div class="home_section_title">
+          <h2>"Hello! Sharing world"</h2>
+          <h1>NEED TO VISIT RWANDA? JOIN THE GORILLA NAMING CEREMONY.</h1>
+        </div>
+        <article>
+          <h4>
+            Kwita Izina is inspired by the ancient Rwandan Tradition of naming a
+            baby soon after its birth. During this ceremony, infant mountain
+            gorillas that were born in Rwanda during the previous year receive
+            their names as well as one newly formed family.
+          </h4>
+        </article>
+        <div class="info">
+          <p>
+            Please contact us per Email for more informations about the gorilla
+            naming ceremony.
+          </p>
+          <div class="link">
+            <a href="#">kwitaizina@volcanoes.rw</a>
+          </div>
+        </div>
+      </section>
+      <section id="Program" class="overView d-flex flex-center">
+        <h2>Gorilla Naming Ceremony Overview</h2>
+        <p>
+          Kwita Izina is a Ceremony that happens every year, and Hundreds of
+          people from different locations are welcomed in Rwanda for that
+          ceremony.
+        </p>
+        <div class="logo">
+          <img src="./images/main-logo.png" alt="" />
+        </div>
+        <div class="line"></div>
+      </section>
+
+      <section id="Team" class="gorilla">
+        <h2>Main Ceremony's Activities.</h2>
+        <p>
+          Welcome to the Gorilla Naming Ceremony! Join us to become a part of us
+          in forming gorilla families by giving name to the born babies. You
+          will enjoy hiking and playing with gorilla's babies.
+        </p>
+        <ul class="image-list d-flex flex-center">
+          <li>
+            <img src="images/picture1.jpg" alt="Gorilla image" />
+            <p class="text">
+              2020 <br />
+              Volcanoes Hiking
+            </p>
+            <span class="backgrd"></span>
+          </li>
+          <li>
+            <img src="images/picture2.jpg" alt="Gollira image" />
+            <p class="text">
+              2021 <br />
+              Gorilla Families
+            </p>
+            <span class="backgrd"></span>
```

Comment:
- [ ] (OPTIONAL) Kindly remove the extra grey background color of this section as the background color in the original design is supposed to be white 👍 
- Your design:
![Annotation 2022-04-22 040228](https://user-images.githubusercontent.com/85113567/164589553-89c8d775-1e1d-4094-8e09-221d1ea43574.jpg)
- Required design:
![Annotation 2022-04-22 040247](https://user-images.githubusercontent.com/85113567/164589568-314c7bcb-4b4d-4d46-b9fb-cc1e18e4e3e2.jpg)

Reply:
Thanks @Kingstalux  I will work on them